config/metasrv.example.toml

```diff
@@ -99,7 +99,12 @@ provider = "raft_engine"
 ## The broker endpoints of the Kafka cluster.
 broker_endpoints = ["127.0.0.1:9092"]
 
-## Number of topics to be created upon start.
+## Create topics for WAL.
+## Set to `true` to automatically create topics for WAL.
+## Otherwise, use topics named `topic_name_prefix_[0..num_topics)`
+create_topic = true
```

Comment:
`auto_create_topics` would be better.

Reply:
Resolved in [b324e5c](https://github.com/GreptimeTeam/greptimedb/pull/4616/commits/b324e5c44b613203d64cd0966c0d927fd74beb8b)